frontend/packages/topology/src/elements/BaseGraph.ts

```diff
@@ -238,6 +248,19 @@ export default class BaseGraph<E extends GraphModel = GraphModel, D = any> exten
     if ('layout' in model) {
       this.setLayout(model.layout);
     }
+    const zoomExtents: [number, number] = [this.zoomExtents[0], this.zoomExtents[1]];
+    let updateExtents = false;
+    if ('minZoom' in model && typeof model.minZoom === 'number') {
```

Comment:
I'm a little taken-back by the usage of the type and this check. I agree it's good to be safe, but I wonder if we are crutching at the wrong spot? Can you make the model setter a little more stable so it's either undefined or a number?